src/options/pack.rs

```diff
@@ -32,6 +34,9 @@ impl Options {
             ap.refer(&mut opt.file)
                 .add_option(&["-f", "--file"], ParseOption,
                     "File to store tar archive at");
+            ap.refer(&mut opt.compression_type)
+                .add_option(&["-t", "--compression-type"], ParseOption,
```

Comment:
Probably the short argument `-t`. If we will ever need a short argument for this option, I would prefer `tar`-compatible options `-zJj`